src/snowcli/cli/project/util.py

```diff
@@ -60,3 +61,15 @@ def first_set_env(*keys: str):
 
 def get_env_username() -> Optional[str]:
     return first_set_env("USER", "USERNAME", "LOGNAME")
+
+
+def is_mount(path: Path):
+    """
+    A wrapper around Path.is_mount() that does not throw an exception
+    on Windows. Instead, we treat Windows as if it has no mount points.
+    On POSIX platforms, behaviour is identical to Path.is_mount().
+    """
+    try:
+        return path.is_mount()
+    except NotImplementedError:
+        return False
```

Comment:
Why not assume positive scenario?

Reply:
It doesn't really matter for our use case; we just use it as a stopping condition if it changes from the value of the initial directory. If you'd rather it default to `True` when not implemented, I'm fine with that.

I looked quickly to see if I could find the source code to understand how the Windows one works but my quick timebox didn't lead to the sourcecode :)